optimize/client/src/components/Reports/controlPanels/DefinitionList/RenameVariablesModal.scss

```diff
@@ -1,6 +1,9 @@
+@use '@carbon/styles/scss/spacing' as *;
+
 .RenameVariablesModal {
-  .cds--modal-container {
+  .ModalContainer {
```

Comment:
Is there a reason for the difference casing on this class name? If not I would suggest you match the other ones using camel case by renaming it to `modalContainer`.

Reply:
Good catch!
I think i did not by mistake